packages/venia-ui/lib/components/CheckoutPage/PaymentInformation/summary.js

```diff
@@ -119,36 +65,13 @@ const Summary = props => {
                             defaultMessage={'Payment Information'}
                         />
                     </h5>
-                    <LinkButton
-                        className={classes.edit_button}
-                        onClick={onEdit}
-                        type="button"
-                    >
-                        <Icon
-                            size={16}
-                            src={EditIcon}
-                            classes={{ icon: classes.edit_icon }}
-                        />
-                        <span className={classes.edit_text}>
-                            <FormattedMessage
-                                id={'global.editButton'}
-                                defaultMessage={'Edit'}
-                            />
-                        </span>
-                    </LinkButton>
                 </div>
                 <div className={classes.card_details_container}>
-                    <span className={classes.payment_type}>
-                        <FormattedMessage
-                            id={'global.creditCard'}
-                            defaultMessage={'Credit Card'}
-                        />
-                    </span>
                     <span className={classes.payment_details}>
-                        {paymentSummary}
+                        {/* TODO: Do we want to display some other info? On mobile this can feel odd since price summary isn't seen until you go to the review step. */}
+                        {selectedPaymentMethod.title}
```

Comment:
This should only be shown if the payment method does not have a summary view right? What info could we show here in the event that there is no custom summary view for the method?

Reply:
@sirugh I don't know what we should do here but yes your right all payment should have own summery but in case you forget to register own would be nice to have a default.

Reply:
I think `title` is as good as we're going to get: https://devdocs.magento.com/guides/v2.4/graphql/queries/cart.html#SelectedPaymentMethod